python/sglang/srt/server.py

```diff
@@ -392,28 +392,29 @@ def _wait_and_warmup(server_args, pipe_finish_writer):
         print(f"Initialization failed. warmup error: {last_traceback}", flush=True)
         sys.exit(1)
 
-    # Send a warmup request
-    try:
-        for _ in range(server_args.dp_size):
-            res = requests.post(
-                url + "/generate",
-                json={
-                    "text": "The capital city of France is",
-                    "sampling_params": {
-                        "temperature": 0,
-                        "max_new_tokens": 8,
+    if not server_args.skip_tokenizer_init:
```

Comment:
let us keep this warmup test, but send token ids instead. We can send token ids `[10, 11, 12]`